chat-plugins/info.js

```diff
@@ -1750,8 +1750,8 @@ exports.commands = {
 		"Adding the parameter 'all' or 'table' will display the information with a table of all type combinations."],
 
 	statcalc: function (target, room, user) {
-		if (!this.canBroadcast()) return;
 		if (!target) return this.parse("/help statcalc");
+		if (!this.canBroadcast()) return;
```

Comment:
Oh, I didn't see the change in the git, now I see that it was moved to another line, ops